File: Python/Chapter03/person.py
class Person:
	def __init__(self, name, konomi):
		self.__name = name
		self.__konomi = konomi
		self.kibun = '普通'
		
	def listentomusic(self, cdplayer):
		genre = cdplayer.playCd()
		if genre == self.__konomi:
			self.kibun = '楽しい' # ジャンル=好み
		else:
			self.kibun = 'イライラ' # ジャンル≠好み

File: Java/Chapter11/Person.java
public class Person {
    private String _name;
    private String _konomi;
    private String _kibun = "普通";
    
    public Person(String name, String konomi) {
        _name = name;
        _konomi = konomi;
    }
    
    public void listenToMusic(CdPlayer cdPlayer) {
	String genre = cdPlayer.playCd();
        if (genre.equals(_konomi))
	    _kibun = "楽しい";
        else 
	    _kibun = "イライラ";
    }

    public String getKibun() {
	return _kibun;
    }
    
    public String getName() {
        return _name;
    }

}
